Create ui-c8y-1021-58-1-conditionally-show-x-509-tab-in-device-details-8552-graft-release.md (#3164)
SoftwareAG/c8y-docs: content/change-logs/device-management/ui-c8y-1021-58-1-conditionally-show-x-509-tab-in-device-details-8552-graft-release.md

---
date: ""

> title: X.509 certificates tab is now shown only on applicable devices

Reviewer: ```suggestion
title: X.509 certificates tab is now only shown for applicable devices
```

> title: Conditionally show "X509" tab in device details (#8552) [GRAFT][release/cd] (#8710)

Reviewer 2: ```suggestion
title: X.509 certificates tab is now shown only on applicable devices
```

product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component--KIsStyzM
    label: Device Management app
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-62971
version: 1021.58.1
---

> Conditionally show "X509" tab in device details (#8552) [GRAFT][release/cd] (#8710)

Reviewer 2: ```suggestion
To prevent user confusion, the "x509" certificates tab is now only shown if the device actually contains at least one X.509 certificate. The behavior of the certificates tab itself remains unchanged and it will continue to display any associated X.509 certificates for a device as before.
```

> To prevent user confusion, the **x509** tab in the device details is only shown if the device contains at least one X.509 certificate. The behavior of the certificates tab itself remains unchanged and it will continue to display any associated X.509 certificates for a device as before.

Reviewer: ```suggestion
To prevent user confusion, the **x509** tab in the device details is now only shown if the device contains at least one x.509 certificate. The behavior of the certificates tab itself remains unchanged and it will continue to display any associated x.509 certificates for a device as before.
```

> To prevent user confusion, the "x509" certificates tab is now only shown if the device actually contains at least one X.509 certificate. The behavior of the certificates tab itself remains unchanged and it will continue to display any associated X.509 certificates for a device as before.

Reviewer: ```suggestion
To prevent user confusion, the **x509** tab in the device details is only shown if the device contains at least one X.509 certificate. The behavior of the certificates tab itself remains unchanged and it will continue to display any associated X.509 certificates for a device as before.
```
